python/pyhail/context.py

```diff
@@ -190,26 +204,70 @@ def import_bgen(self, path, tolerance=0.2, sample_file=None, npartitions=None):
 
         return self.run_command(None, pargs)
 
-    def import_gen(self, path, tolerance=0.2, sample_file=None, npartitions=None, chromosome=None):
-        """Import .bgen files as VariantDataset
+    def import_gen(self, path, sample_file=None, tolerance=0.02, npartitions=None, chromosome=None):
+        """Import .gen files as VariantDataset
+
+        **Examples**
+
+        To read a .gen file and a .sample file and write to a .vds file::
+
+        >>> (hc.import_gen('data/example.gen', sample_file='data/example.sample')
+        >>>  .write('data/example.vds'))
+
+        To load multiple files at the same time, use `Hadoop glob patterns <../reference.html#hadoopglob>`_::
+
+        >>> (hc.import_gen('data/example.chr*.gen', sample_file='data/example.sample')
+        >>>  .write('data/example.vds'))
+
+        **Notes**
+
+        For more information on the .gen file format, see `here <http://www.stats.ox.ac.uk/%7Emarchini/software/gwas/file_format.html#mozTocId40300>`_.
+
+        To ensure that the .gen file(s) and .sample file are correctly prepared for import:
+
+        - Files should reside in the Hadoop file system
```

Comment:
Don't say they need to be in Hadoop.  We don't use Hadoop in the cloud, for example.